lib/client-assets.php

```diff
@@ -262,7 +262,7 @@ function gutenberg_register_packages_scripts( $scripts ) {
 		// Add dependencies that cannot be detected and generated by build tools.
 		switch ( $handle ) {
 			case 'wp-block-library':
-				array_push( $dependencies, 'editor' );
+				array_push( $dependencies, 'wp-dom-ready' );
```

Comment:
Could you explain this change a bit more? I'm curious why we need to remove editor but add something else!

Reply:
The old `editor` script no longer exists so we can't have that as a dependency. If we don't change this to `wp-dom-ready` then Gutenberg doesn't load at all.